docs/sources/datasources/tempo.md

```diff
@@ -204,20 +204,23 @@ Click on the service to see a context menu with additional links for quick navig
 
 ## APM table
 
-The APM (Application Performance Management) table allows you to view several APM metrics out of the box.
+The Application Performance Management (APM) table lets you view several APM metrics out of the box.
+The APM table is part of the APM dashboard.
+For more information, refer to the [APM dashboard documentation](https://grafana.com/docs/tempo/next/metrics-generator/app-performance-mgmt/).
 
 To display the APM table:
 
-1. Activate the tempoApmTable feature flag in your ini file.
-1. Link a Prometheus datasource in the Tempo datasource settings.
+1. Activate the `tempoApmTable` feature flag in your Grafana `ini` file.
```

Comment:
Why is ini being code-formatted here? Should this be `grafana.ini`, or more generically "Grafana configuration file" per https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#configuration-file-location ?

Reply:
Thank you. I've updated the formatting. 
```suggestion
1. Activate the `tempoApmTable` feature flag in your `grafana.ini` file.
```